src/SFML/Window/DRM/DRMContext.cpp

```diff
@@ -0,0 +1,606 @@
+////////////////////////////////////////////////////////////
+//
+// SFML - Simple and Fast Multimedia Library
+// Copyright (C) 2020 Andrew Mickelson
+//               2013 Jonathan De Wachter (dewachter.jonathan@gmail.com)
+//
+// This software is provided 'as-is', without any express or implied warranty.
+// In no event will the authors be held liable for any damages arising from the use of this software.
+//
+// Permission is granted to anyone to use this software for any purpose,
+// including commercial applications, and to alter it and redistribute it freely,
+// subject to the following restrictions:
+//
+// 1. The origin of this software must not be misrepresented;
+//    you must not claim that you wrote the original software.
+//    If you use this software in a product, an acknowledgment
+//    in the product documentation would be appreciated but is not required.
+//
+// 2. Altered source versions must be plainly marked as such,
+//    and must not be misrepresented as being the original software.
+//
+// 3. This notice may not be removed or altered from any source distribution.
+//
+////////////////////////////////////////////////////////////
+
+////////////////////////////////////////////////////////////
+// Headers
+////////////////////////////////////////////////////////////
+#include <SFML/Window/DRM/DRMContext.hpp>
+#include <SFML/Window/DRM/WindowImplDRM.hpp>
+#include <SFML/OpenGL.hpp>
+#include <SFML/System/Err.hpp>
+#include <SFML/System/Sleep.hpp>
+#include <drm-common.h>
+#include <cstdlib>
+#include <cstring>
+#include <errno.h>
+#include <poll.h>
+#include <unistd.h>
+
+namespace
+{
+    static bool initialized = false;
+    static struct drm my_drm;
+    static drmEventContext my_evctx;
+    static pollfd my_fds;
+    static struct gbm_device *my_gbm_device = NULL;
+    static int context_count = 0;
+    static EGLDisplay display = EGL_NO_DISPLAY;
+    static int waiting_for_flip = 0;
```

Comment:
`static` is redundant in an anonymous namespace.

Reply:
`*` and `&` are aligned towards the type in SFML's style. This applies to all other lines as well.